tapir/coop/templates/coop/pdf/extra_shares_confirmation_pdf.default.html

```diff
@@ -0,0 +1,83 @@
+{% load static %}
+{% load django_bootstrap5 %}
+{% load accounts %}
+{% load i18n %}
+
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>
+        {% blocktranslate with display_name=member_infos.get_display_name %}
+            Confirmation acquisition of additional shares in {{ coop_name }} for {{ display_name }}
+        {% endblocktranslate %}
+    </title>
+    <link rel="stylesheet" href="{% static 'coop/css/membership_agreement_pdf.css' %}">
+</head>
+
+<body>
+
+<!-- Falzmarken -->
+<span style="border-top: 1px solid grey; width: 5mm; position: fixed; left: -15mm; top: 60mm;"></span>
+<span style="border-top: 1px solid grey; width: 5mm; position: fixed; left: -15mm; top: 165mm;"></span>
+
+<!-- A DIN 5008 Letter has 2,7cm of top margin -->
+<div style="display: flex">
+    <div style="margin-top: 17.7mm; margin-bottom: 30mm; margin-right: 30mm;">
+        <span style="font-size: 6pt;">
+            {{ member_infos.get_display_name }},<br/>
+            {{ member_infos.street }},<br/>
+            {% if member_infos.street_2 %} {{ member_infos.street_2 }},<br/>{% endif %}
+            {{ member_infos.postcode }} {{ member_infos.city }}
+        </span>
+        <div style="margin-top: 2mm;">
+            SuperCoop Berlin eG<br/>
+            Oudenarder Straße 16<br/>
+            13347 Berlin
```

Comment:
Ideally, we'd have a base repo that would not be this one and that would only have the default files. 
Since we currently don't have that separate base repo, we sort of use this repo as base. So I would put our stuff in the .default. files to avoid creating the not default ones and facilitate merges for other forks. 
As soon as we have a separate base repo, we should use the defaults from that repo and put our custom templates in the not default files.

But! For that one case you cited, since you added the address to the settings, I need to merge that in that PR and use the adress from the settings.